### DISCUSSION TITLE
Experienced an issue on running the strapi-admin module.

### DISCUSSION THREAD

**Discussion by sasicodes** [strapi/strapi#6477] [Legacy Topics]
Issue:

I took Fork from GitHub and tried to run the strapi-admin module in the local machine. The prerequisites were fulfilled.

When I run yarn develop its running in localhost:4000/admin, opened it in browser buts its keep loading, nothing on the screen.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by beltranrengifo** [comment]
@sasidharank here with the same problem, could you solve it?

**Reply by sasicodes** [reply]
Nope
